pkg/distro/fedora/distro.go

```diff
@@ -487,6 +484,32 @@ func mkMinimalRawImgType(d distribution) imageType {
 	return it
 }
 
+// mkNetinstISOImgType adds a netinst image type that creates an Anaconda boot.iso
+func mkNetinstISOImgType(d distribution) imageType {
+	return imageType{
+		name:        "netinst",
```

Comment:
I suppose we could call this `everything-netinst` to follow more closely what 'variant' it belongs to; I tried to do some work around this in #1233 so it might be nice to do this per-emptively here as well.

Reply:
Our support for variants doesn't really do much, and the boot.iso isn't tied to a specific variant, I just picked 'Everything' because 'Unknown' seems wrong to expose to users. Maybe keep netinst as an alias?